packages/compass-aggregations/src/components/pipeline-results-workspace/pipeline-results-header.tsx

```diff
@@ -14,8 +18,8 @@ type PipelineResultsHeaderProps = {
   onChangeResultsView: (viewType: ResultsViewType) => void;
   resultsViewType: ResultsViewType;
   isMergeOrOutPipeline: boolean;
-  onChangePipelineOutputOption: (val: PipelineOutputOption) => void;
-  pipelineOutputOption: PipelineOutputOption;
+  expandPipelineResults: () => void;
```

Comment:
Ditto naming (especially because the component was already using correct naming scheme, now we're just mixing the two, why?)